src/main/java/org/jfrog/hudson/pipeline/common/types/deployers/MavenDeployer.java

```diff
@@ -102,4 +114,44 @@ private static MavenDeployer createEmptyDeployer() {
         dummy.setThreads(1);
         return dummy;
     }
+
+    public static void addDeployedMavenArtifactsActionFromModules(Run build, String artifactoryUrl, List<Module> modules) {
```

Comment:
Let's add a description, explaining what this method does.
addDeployedMavenArtifactsActionFromModules --> addDeployedArtifactsActionFromModules